addons/web/static/src/js/views/action_model.js

```diff
@@ -33,6 +33,14 @@ odoo.define("web/static/src/js/views/action_model.js", function (require) {
      */
     class ActionModel extends Model {
 
+        constructor(extensions, globalConfig, importedState) {
+            super(
+                Object.assign(extensions, { Search: {} }),
```

Comment:
I would use reverse order in case search extension is changed in futur

Reply:
Done